src/PixiParser/Helpers.cs

```diff
@@ -1,19 +1,20 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Drawing;
```

Comment:
The parser should use primitives, such as a List of bytes. PixiEditor doesn't even use System.Drawing.Color, but System.Windows.Media.Color. But it's not cross-platform, so just use bytes instead